tools/c7n_azure/c7n_azure/resources/storage.py

```diff
@@ -64,156 +65,110 @@ class resource_type(ArmResourceManager.resource_type):
 
 
 @Storage.action_registry.register('set-firewall-rules')
-class StorageSetNetworkRulesAction(AzureBaseAction):
-    """ Set Network Rules Action
+class StorageSetFirewallAction(SetFirewallAction):
+    """ Set Firewall Rules Action
 
-    Updates Azure Storage Firewalls and Virtual Networks settings.
+     Updates Azure Storage Firewalls and Virtual Networks settings.
```

Comment:
We probably should think on how to extend base-filter comment in the inherited objects to avoid this comments duplication..

Reply:
They were more different than similar in the end, I thought it was better for such complicated filters to have customized comments with accurate samples/links/etc.

Reply:
Yeah, this is not only about this PR :)